tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+
+class TestBugzillaLite(GaiaTestCase):
+    def test_login_BugzillaLite(self):
+        self.BugzillaLite = BugzillaLite(self.marionette)
```

Comment:
Nit: We can rename `BugzillaLite` to `bugzilla_lite`. It's a standard way to spell objects in Python.
We can also remove the `self` because we don't use this variable outside of this function.